bin/gaffer

```diff
@@ -268,6 +268,30 @@ if [[ -n $DELIGHT ]] ; then
 
 fi
 
+# Set up 3rd Party extensions
+##########################################################################
+
+for extension in `echo $GAFFER_EXTENSION_PATHS | sed 's/:/\n/g'`; do
+
+	echo Loading extension: $extension
+
+	if [[ `uname` = "Linux" ]] ; then
+		appendToPath "$extension/lib" LD_LIBRARY_PATH
+	else
+		appendToPath "$extension/lib" DYLD_LIBRARY_PATH
+	fi
+
+	appendToPath "$extension/bin" PATH
+	appendToPath "$extension/python" PYTHONPATH
+	appendToPath "$extension/startup" GAFFER_STARTUP_PATHS
+	appendToPath "$extension/apps" GAFFER_APP_PATHS
+	appendToPath "$extension/graphics" GAFFERUI_IMAGE_PATHS
+	appendToPath "$extension/glsl" IECOREGL_SHADER_PATHS
+	appendToPath "$extension/glsl" IECOREGL_SHADER_INCLUDE_PATHS
+	appendToPath "$extension/shaders" OSL_SHADER_PATHS
```

Comment:
I was wondering if we shouldn't call this subfolder "osl" to sit alongside "glsl"?

Reply:
Sounds good.

Reply:
Should I change the native shaders folder as well or leave it be?

Reply:
It'd be nice to have it all matching I think, so if you don't hit any obstacles let's change it as well, in a separate commit (probably shouldn't backport something that changes directory layout).

Reply:
I'm going to back out of this one because I'm unsure of the implications. I'll leave a todo in the wrapper.